docs/source/get-involved/gsoc/guidelines.md

```diff
@@ -0,0 +1,131 @@
+# GSoC NIU Contributor Application Guidelines
+
+## General tips
+These tips are largely based on the [OpenAstronomy guidelines](https://openastronomy.org/gsoc/student_guidelines.html), with adaptations for the NIU organisation.
+
+1. **Get in touch with the community**
+
+    Open source work is done and communicated in public - the idea here is to demonstrate that you can do this! Join our Zulip, or browse through our GitHub repositories. Listen, ask questions, get to know the people involved, and participate in discussions. 
```

Comment:
Maybe worth linking to the zulip & github too?

Reply:
This might also be a good place to outline clearly that the NIU includes BrainGlobe, and that projects are across both GitHub orgs

Reply:
cool, I'll also clarify this in the GSoC home page